examples/lib/defaultHtmlStepUi.js

```diff
@@ -34,9 +34,11 @@ function DefaultHtmlStepUi(_sequencer, options) {
       '</i></p></div>\
     </div>\
     </form>\
-    <div class="col-md-8 cal">\
-    <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
-    <a><img alt="" style="max-width=100%" class="img-thumbnail step-thumbnail"/></a>\
+    <div class="col-md-8 cal crop-column">\
+      <div class="load" style="display:none;"><i class="fa fa-circle-o-notch fa-spin"></i></div>\
+      <div class = "crop-image">
```

Comment:
Please don't keep any gaps in between the equal to sign and the rest of the code.

Reply:
Sorry..I have a habit of leaving spaces. I've made the changes